data/ecosystem/adopters.yaml

```
@@ -14,6 +14,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: eBay
+  url: https://www.ebay.com
+  components: [Collector]
+  reference: /blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/
+  contact: 'https://github.com/vjsamuel'
```

Nit: I don't think that you need quotes here:

```suggestion
  contact: https://github.com/vjsamuel
```